src/main/core/scheduler/mod.rs

```diff
@@ -5,7 +5,7 @@ pub use thread_per_core::ThreadPerCoreSched;
 pub use thread_per_host::ThreadPerHostSched;
 
 mod logical_processor;
-mod pools;
+pub mod pools;
```

Comment:
Did this end up being needed for the doc tests to run?

Reply:
It isn't needed for them to run, but it's needed so that they will compile, otherwise `use shadow_rs::core::scheduler::pools::bounded::*;` won't be able to access the types we need in the test.

Reply:
Ahh, right makes sense